### PR TITLE
[Botskills] Improve --noRefresh argument description

### PR DESCRIPTION
**Pull request by DiegoCardozo94**
### Purpose
*What is the context of this pull request? Why is it being done?*
Clarify the `--noRefresh` description in order to be more specific related to the functionality of this argument.

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
Change the `--noRefresh` argument description in the connect, disconnect and update commands.

### Tests
*Is this covered by existing tests or new ones? If no, why not?*
*-*
### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*
*-*
#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: tools/botskills/docs/commands/connect.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ For Skills that require other Authentication connection configuration please fol
 | -r, --remoteManifest \<url>   | URL to remote Skill Manifest                                                                                                                                                        |
 | --cs                          | Determine your Virtual Assistant project structure to be a csharp-like structure                                                                                                    |
 | --ts                          | Determine your Virtual Assistant project structure to be a TypeScript-like structure                                                                                                |
-| --noRefresh [true\|FALSE]     | (OPTIONAL) Determine whether the model of your skills connected are not going to be refreshed (by default they are refreshed)                                                       |
+| --noRefresh [true\|FALSE]     | (OPTIONAL) Determine if your skills models connected to the Virtual Assistant are not going to be refresh (by default they are refreshed)                                                 |
```
Comment (matiasroldan6) [inline tools/botskills/docs/commands/connect.md:51]: `to be refresh` -> `to be refreshed` maybe?

Comment (matiasroldan6) [reply tools/botskills/docs/commands/connect.md:51]: what's with the backslash after `true` here? Is that correct?

Comment (DiegoCardozo94) [reply tools/botskills/docs/commands/connect.md]: If I remove the backslash the text dissapears from the description column

File: tools/botskills/docs/commands/connect.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ For Skills that require other Authentication connection configuration please fol
 | -r, --remoteManifest \<url>   | URL to remote Skill Manifest                                                                                                                                                        |
 | --cs                          | Determine your Virtual Assistant project structure to be a csharp-like structure                                                                                                    |
 | --ts                          | Determine your Virtual Assistant project structure to be a TypeScript-like structure                                                                                                |
-| --noRefresh [true\|FALSE]     | (OPTIONAL) Determine whether the model of your skills connected are not going to be refreshed (by default they are refreshed)                                                       |
+| --noRefresh [true\|FALSE]     | (OPTIONAL) Determine whether or not the skills models connected to the Virtual Assistant are not going to be trained and published in an existing dispatch model (by default they are trained)                                                 |
```
Comment (LCarbonellRod) [inline tools/botskills/docs/commands/connect.md:51]: In the `are not going to be` senctence I think that the `not` is redundant as you are using `whether or not`.

File: tools/botskills/docs/commands/disconnect.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ botskills disconnect [options]
 | -i, --skillId \<id>           | Id of the skill to remove from your Virtual Assistant (case sensitive)                                                                                                      |
 | --cs                          | Determine your Virtual Assistant project structure to be a csharp-like structure                                                                                            |
 | --ts                          | Determine your Virtual Assistant project structure to be a TypeScript-like structure                                                                                        |
-| --noRefresh                   | (OPTIONAL) Determine whether the model of your skills connected are not going to be refreshed (by default they are refreshed)                                               |
+| --noRefresh                   | (OPTIONAL) Determine whether or not the skills models connected to the Virtual Assistant are not going to be trained and published in an existing dispatch model (by default they are trained)                                             |
```
Comment (LCarbonellRod) [inline tools/botskills/docs/commands/disconnect.md:20]: Same comment as in the connect.md file

File: tools/botskills/docs/commands/update.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ botskills update [options]
 | -r, --remoteManifest \<url>   | URL to remote Skill Manifest                                                                                                                                                |
 | --cs                          | Determine your Virtual Assistant project structure to be a csharp-like structure                                                                                            |
 | --ts                          | Determine your Virtual Assistant project structure to be a TypeScript-like structure                                                                                        |
-| --noRefresh [true\|FALSE]     | (OPTIONAL) Determine whether the model of your skills connected are not going to be trained (by default they are trained)                                                   |
+| --noRefresh [true\|FALSE]     | (OPTIONAL) Determine whether or not the skills models connected to the Virtual Assistant are not going to be trained and published in an existing dispatch model (by default they are trained)                                             |
```
Comment (LCarbonellRod) [inline tools/botskills/docs/commands/update.md:21]: Same comment as in the connect.md file

File: tools/botskills/src/botskills-connect.ts [hunk 5]
```diff
@@ -35,7 +35,7 @@ program
     .option('-r, --remoteManifest <url>', 'URL to remote Skill Manifest')
     .option('--cs', 'Determine your assistant project structure to be a CSharp-like structure')
     .option('--ts', 'Determine your assistant project structure to be a TypeScript-like structure')
-    .option('--noRefresh [true|FALSE]', '[OPTIONAL] Determine whether the model of your skills connected are not going to be refreshed (by default they are refreshed)')
+    .option('--noRefresh [true|FALSE]', '[OPTIONAL] Determine whether or not the skills models connected to the Virtual Assistant are not going to be trained and published in an existing dispatch model (by default they are trained)')
```
Comment (LCarbonellRod) [inline tools/botskills/src/botskills-connect.ts:38]: Same comment as in the connect.md file

File: tools/botskills/src/botskills-disconnect.ts [hunk 6]
```diff
@@ -32,7 +32,7 @@ program
     .option('-i, --skillId <id>', 'Id of the skill to remove from your assistant (case sensitive)')
     .option('--cs', 'Determine your assistant project structure to be a CSharp-like structure')
     .option('--ts', 'Determine your assistant project structure to be a TypeScript-like structure')
-    .option('--noRefresh', '[OPTIONAL] Determine whether the model of your skills connected are not going to be refreshed (by default they are refreshed)')
+    .option('--noRefresh [true|FALSE]', '[OPTIONAL] Determine whether or not the skills models connected to the Virtual Assistant are not going to be trained and published in an existing dispatch model (by default they are trained)')
```
Comment (LCarbonellRod) [inline tools/botskills/src/botskills-disconnect.ts:35]: Same comment as in the connect.md file

File: tools/botskills/src/botskills-update.ts [hunk 7]
```diff
@@ -34,7 +34,7 @@ program
     .option('-r, --remoteManifest <url>', 'URL to remote Skill Manifest')
     .option('--cs', 'Determine your assistant project structure to be a CSharp-like structure')
     .option('--ts', 'Determine your assistant project structure to be a TypeScript-like structure')
-    .option('--noRefresh [true|FALSE]', '[OPTIONAL] Determine whether the model of your skills connected are not going to be trained (by default they are trained)')
+    .option('--noRefresh [true|FALSE]', '[OPTIONAL] Determine whether or not the skills models connected to the Virtual Assistant are not going to be trained and published in an existing dispatch model (by default they are trained)')
```
Comment (LCarbonellRod) [inline tools/botskills/src/botskills-update.ts:37]: Same comment as in the connect.md file

File: tools/botskills/docs/commands/connect.md (hunk 2 above)
Comment (DiegoCardozo94) [reply tools/botskills/docs/commands/connect.md:51]: I think we can remove the first not.

Comment (LCarbonellRod) [reply tools/botskills/docs/commands/connect.md:51]: Like it too!

**Pull request closed without merging**